### ISSUE TITLE
Communcation between Android side and Rust side

### ISSUE THREAD

**Issue by alinawlo** [amosproj/amos2024ws03-android-zero-instrumentation#42]
## User story
1. As a a developer
2. I want / need to establish connection between both sides of the project (Android and Rust).
3. So that data can be transmitted via rust and displayed on the android side

## Acceptance criteria
* Android UI is responsive with Rust side

## Definition of done (DoD)
* Added only after week 5
* Feature is working as expected 
* Feature has been merged into the mainline

## DoD general criteria
* Feature has been fully implemented
* Product owner approved features
* Developers agreed to release
